autosklearn/metalearning/metalearning/meta_base.py

```diff
@@ -65,7 +65,7 @@ def add_dataset(self, name, metafeatures):
             self.metafeatures.drop(name.lower(), inplace=True)
         self.metafeatures = self.metafeatures.append(metafeatures)
 
-        runs = pd.Series([], name=name)
+        runs = pd.Series([], name=name, dtype=object)
```

Comment:
Looking at this, I'm currently wondering whether this should a) be a dataframe as the other entries in `self.algorithm_runs`, and b) be of type float

Reply:
It's a Series of type float.